daml-foundations/daml-ghc/src/DA/Daml/GHC/Compiler/Convert.hs

```diff
@@ -974,7 +977,13 @@ mkProjBindings env recExpr recTyp vsFlds e =
     , not (isDeadOcc (occInfo (idInfo v)))
     ]
 
--- | Convert casts induced by newtype definitions. The definition
+-- | Convert casts induced by newtype definitions.
+convertCast :: Env -> LF.Expr -> Coercion -> ConvertM LF.Expr
+convertCast env expr co = do
+    (to, _from) <- evalStateT (convertCoercion env co) 0
+    pure (to `ETmApp` expr)
+
+-- Convert a coercion to a pair of lambdas. The definition
```

Comment:
I want to know the types of the Coercion. My guess is that given a Coercion `a ~ b` I get a lambda `a -> b` and `b -> a`? If so, can we get that in the docs.

Reply:
that's correct, I'll put it in the docs.